Tools/CMake/torque3d.cmake

```diff
@@ -645,11 +645,13 @@ else()
   endif()
 endif()
 
-if(NOT EXISTS "${projectOutDir}/${PROJECT_NAME}.torsion")
-    CONFIGURE_FILE("${cmakeDir}/template.torsion.in" "${projectOutDir}/${PROJECT_NAME}.torsion")
-endif()
-if(EXISTS "${CMAKE_SOURCE_DIR}/Templates/${TORQUE_TEMPLATE}/game/main.${TORQUE_SCRIPT_EXTENSION}.in")
-    CONFIGURE_FILE("${CMAKE_SOURCE_DIR}/Templates/${TORQUE_TEMPLATE}/game/main.${TORQUE_SCRIPT_EXTENSION}.in" "${projectOutDir}/main.${TORQUE_SCRIPT_EXTENSION}")
+CONFIGURE_FILE("${cmakeDir}/template.torsion.in" "${projectOutDir}/${PROJECT_NAME}.torsion")
```

Comment:
Fair warning, this would overwrite any changes the developer made to their Torsion project. This might not be desirable..
Could we regex the script extension in, instead? Or possibly have a "cmake.torsionconfig" file or something like that

Reply:
The reason for removing the `if(Exists...` check is because when using cmake gui, the .torsion file gets created on the first configure pass after supplying a project name. This is before the user has had a chance to edit the script extension, so torsion is always setup for the .tscript extension regardless of the value of TORQUE_SCRIPT_EXTENSION. But, I agree, a better solution needs to be found for preventing user changes from getting overwritten.

Reply:
Hmm what about a warning?
I mean, sure probably no one reads the warnings, but it'd be nice with a little warning.